src/GraphControl/Control/Grapher.cpp

```diff
@@ -465,15 +464,8 @@ namespace GraphControl
     vector<Equation ^> Grapher::GetGraphableEquations()
     {
         vector<Equation ^> validEqs;
-
-        for (Equation ^ eq : Equations)
```

Comment:
The change is correct, but because `std::copy_if` provides the same performances than the existing loop, I would suggest not to change this part. The original code is straightforward and easier to understand for contributors with little C++ experience.

Reply:
That's fair.  I've reached a point where I look at loops and see if it's an existing algorithm.  Sometimes I go overboard.  In C#, I probably would have done `Equations.Where(eq => eq.IsGraphableEquation()).ToList()` and that's not even as efficient as a raw loop due to Linq overheads.